Aptoide/appcoins-wallet-android
# [APPC-1211] Add analytics to local payments

**What does this PR do?**

   Added Local Payment analytics. 
Refactored Paypal analytics to avoid sending some of them when doing a local payment and a top up. 
Fixed CC analytics being sent when doing a purchase with paypal

**Where should the reviewer start?**

AdyenAuthorizationPresenter.java
LocalPaymentsPresenter.kt

**How should this be manually tested?**

Do a purchase with PayPal and Credit card (IaP and popup) and check if the events are being sent correctly on the console.
The local payments events will be later tested with Tiago. The only one we are able to check is the payment methods detail one and the payment event for the assync payments (alfamart)

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-1211

**Code Review Checklist**
